pkg/network/http/ebpf_main.go

```diff
@@ -45,6 +46,8 @@ const (
 	batchNotificationsChanSize = 100
 
 	probeUID = "http"
+
+	maxRequestLinger = 30 * time.Second
```

Comment:
I don't remember the exact semantics in eBPF, but is there any danger of removing an entry that might get matched up when the next `/connections` query is done? Like do we need to wait at least `2*x` (where `x` is connection interval time), to ensure request/responses from two adjacent intervals have a chance to match?